src/Form.php

```diff
@@ -143,6 +143,47 @@ protected function initLayout()
         $this->buttonSave->on('keypress', new jsExpression('if (event.keyCode === 13){$([name]).form("submit");}', ['name' => '#'.$this->name]));
     }
 
+    /**
+     * Add a View into the form layout.
+     * The newly added view or a sub view of the newly added view
+     * can later be used to add form field to it.
+     *
+     * @param string|array|View $view
+     * @param bool              $hasDivider
+     *
+     * @throws Exception
+     *
+     * @return View
+     */
+    public function addLayoutView($view, $hasDivider = true)
+    {
+        if ($this->layout === null) {
+            throw new Exception('Layout need to be initialized prior to add View.');
+        }
+
+        $v = $this->layout->add($view);
+        if ($hasDivider) {
+            $this->layout->add(['ui' => 'hidden divider']);
```

Comment:
this makes assumption about layout of the form.

how about Form_Layout->addSection() there ?

also, technically calling form->add(something) and have it inserted into layout is confusing.